marklogic-data-hub-central/ui/e2e/cypress/integration/login/authorization.spec.tsx

```diff
@@ -169,10 +169,11 @@ describe("login", () => {
     loadPage.addToNewFlow(stepName).click("top", {force: true});
     runPage.newFlowModal().should("not.exist");
     loadPage.existingFlowsList(stepName).click({force: true});
-
-    cy.log("**Clicks on Delete button**");
-    loadPage.deleteStep(stepName).should("be.visible").click();
-    loadPage.deleteConfirmation("No").click();
+    loadPage.addToNewFlow(stepName).click("top", {force: true}).then(() => {
```

Comment:
Why do we need to open and close the modal again?
Delete step should be visible already at the current step of the flow. Maybe scroll into view if the problem is that is not in the screen?

Reply:
Hey. The issue it's that the tooltip appears and it's on top of the delete button. You cannot scroll into view that element, because the reason why it's not visible it's that we make that tooltip appear to check we the role doesn't have permissons and we don't do anything to make it go away before clicking the delete button.